package.json

```
@@ -270,7 +270,7 @@
     "sass-loader": "^7.0.1",
     "selenium-webdriver": "^3.5.0",
     "sesify": "^4.2.1",
-    "sesify-viz": "^2.0.1",
+    "sesify-viz": "3.0.5",
```

```suggestion
    "sesify-viz": "^3.0.5",
```
Could this be left as a range, or does it need to be pinned here?

Maybe less important given that this is a tool that's used independently, but as a general rule ranges allow for better de-duplication and make life easier for downstream projects.

---

yeah yarn gave me this when I specified a version (to make sure it updated). I don't think npm behaved this way.